src/App.css

```diff
@@ -0,0 +1,100 @@
+body {
+	display: flex;
+	justify-content: center;
+	margin-bottom: 100px;
+}
+
+img {
+	align-items: center;
+	justify-content: center;
+}
+input {
+	border-top: none;
+	border-left: none;
```

Comment:
I feel as if you can literally just do `border: none;` but I could be wrong, it would just make your life a bit easier